### ISSUE TITLE
User Build - Unable to upload file to uss with Zowe profile ZOSMF (Application hang on validating z/OSMF profile) - Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.

### ISSUE THREAD

**Issue by FALLAI-Denis** [IBM/zopeneditor-about#245]
## Development environment used

- Z Open Editor version: 2.0.3
- Editor Platform
  - [X] Visual Studio Code
  - [ ] Red Hat CodeReady Workspaces
  - [ ] Eclipse Che
  - [ ] Standalone Theia
- Editor Platform Version: 1.67.2
- Operating System: Windows 10 x64
- Java Version: 1.8
- [ ] Related to RSE API?
  - RSE API Plugin version:
  - Zowe CLI version:
  - Node.js version:
- Logs attached: yes

[13-IBM Z Open Editor.log](https://github.com/IBM/zopeneditor-about/files/8757010/13-IBM.Z.Open.Editor.log)
[zowe.vscode-extension-for-zowe-2.0.2logs - imperative (extract).log](https://github.com/IBM/zopeneditor-about/files/8757023/zowe.vscode-extension-for-zowe-2.0.2logs.-.imperative.extract.log)
[zowe.vscode-extension-for-zowe-2.0.2logs - zowe (extract).log](https://github.com/IBM/zopeneditor-about/files/8757027/zowe.vscode-extension-for-zowe-2.0.2logs.-.zowe.extract.log)

## Problem Description

New problem after #235.

When activating an IBM User Build command, Z Open Editor freezes for a very very very long time on testing the z/OSMF profile, then the "IBM User Build" output shows `Stopping User Build. An error occurred uploading required files: Timed out waiting for password.`, (no password is required during freeze).

The issue appeared after **migrating** from Zowe CLI V1 to Zowe CLI V2.

The `zowe.config.json` file containing only the necessary profiles is present in the root of the VS Code workspace, and another `zowe.config.json` file, with more profiles, is declared in the home directory of the user.

### Observed behavior

IBM User Build output:
```
This log file is available at c:\Users\S0070188\AppData\Roaming\Code\logs\20220523T190334\exthost1\IBM.zopeneditor\userbuild-20220523-172152.log
Welcome to IBM User Build. Run User Build from the pop-up menu in an open editor with a COBOL, PL/I, or HLASM program.
Visit our IBM Docs to learn about IBM User Build here: https://www.ibm.com/docs/en/z-modernization-stack/2022.1.1?topic=code-building-cobol-pli-hlasm-programs-user-build.
=====================
Running User Build...
Identified the local root workspace path as c:\Trv\DepotsGIT\environn-techleads-sandbox
---------------------
Checking the ZAPP file or workspace settings...
DBB ZAPP or workspace settings should be defined in either your repository's zapp.yaml, zapp.json or .vscode/settings.json file.
    command: /bin/echo
    buildScriptArgs: --userBuild,--workspace /u/J070188/vscode,--application environ-techleads-sandbox/src,--hlq J070188.VSCODE.USERBUILD,--outDir /u/J070188/vscode/logs
    buildScriptPath: /tmp
    additionalDependencies: application-conf
---------------------
Checking user settings...
User settings should be defined in your VS Code or Theia Settings under zopeneditor.userbuild.userSettings
    localWorkspacePath: **Optional setting is missing**
    dbbWorkspace: /u/J070188/vscode
    dbbHlq: J070188.VSCODE.USERBUILD
    dbbLogDir: /u/J070188/vscode/logs
Using .gitattributes file at c:\Trv\DepotsGIT\environn-techleads-sandbox\.gitattributes
---------------------
The default Zowe CLI profile specified as GMVS in zopeneditor.zowe setting. Will try loading that.
Using Zowe profile GMVS
Stopping User Build. An error occurred uploading required files: Timed out waiting for password.
```

Message found in the log `C:\Users\<user>\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\logs\zowe.log`:
```
[2022/05/23 19:21:51.781] [DEBUG] [extension.js:32] [r [Error]: An error occurred collecting all configurations from the profile root directory "C:\Users\S0070188\.zowe\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.
	at new d (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:269457)
	at new c (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:261507)
	at new f (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:80679)
	at y.getCliProfileManager (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:52:68839)
	at y.getSchema (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:52:67470)
	at Function.getZoweVersion (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1121587)
	at Function.<anonymous> (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1020213)
	at Generator.next (<anonymous>)
	at s (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1014409)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)] {
  mDetails: {
    msg: 'An error occurred collecting all configurations from the profile root directory "C:\\Users\\S0070188\\.zowe\\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.',
    additionalDetails: [r [Error]: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.
    	at Function.crashInTeamConfigMode (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:97792)
    	at Function.getAllProfileDirectories (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:96536)
    	at f.collectAllConfigurations (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:286781)
    	at new d (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:269211)
    	at new c (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:261507)
    	at new f (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:72:80679)
    	at y.getCliProfileManager (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:52:68839)
    	at y.getSchema (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.2\out\src\extension.js:52:67470)
    	at Function.getZoweVersion (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1121587)
    	at Function.<anonymous> (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1020213)
    	at Generator.next (<anonymous>)
    	at s (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1014409)
    	at processTicksAndRejections (node:internal/process/task_queues:96:5)] {
      mDetails: [Object],
      mMessage: 'Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.'
    }
  },
  mMessage: 'An error occurred collecting all configurations from the profile root directory "C:\\Users\\S0070188\\.zowe\\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.'
}
[2022/05/23 19:21:52.099] [DEBUG] [extension.js:32] File was saved -- determining whether the file is a USS file or Data set.
 Comparing (case insensitive) c:\Users\S0070188\AppData\Roaming\Code\User\settings.json against directory C:\Trv\DepotsGIT\Temp\temp\_D_andC:\Trv\DepotsGIT\Temp\temp\_U_
[2022/05/23 19:21:52.104] [DEBUG] [extension.js:32] File c:\Users\S0070188\AppData\Roaming\Code\User\settings.json is not a data set or USS file 
[2022/05/23 19:21:53.068] [DEBUG] [extension.js:32] File was saved -- determining whether the file is a USS file or Data set.
 Comparing (case insensitive) c:\Users\S0070188\AppData\Roaming\Code\User\settings.json against directory C:\Trv\DepotsGIT\Temp\temp\_D_andC:\Trv\DepotsGIT\Temp\temp\_U_
[2022/05/23 19:21:53.073] [DEBUG] [extension.js:32] File c:\Users\S0070188\AppData\Roaming\Code\User\settings.json is not a data set or USS file 
```

![image](https://user-images.githubusercontent.com/63861965/169877649-b2edc2db-c90c-4a65-ad01-4c30c68c97ee.png)

**Comment by FALLAI-Denis** [comment]
For information, same message error on deprecated zowe cli command:

```
>zowe profiles set zosmf gmvs
Error occurred while setting default profile for zosmf.
Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.
```

**Comment by FALLAI-Denis** [comment]
Hi @phaumer,

I did some new tests:
* rollback on the Z Open Editor and Zowe Explorer versions: problem still present
* rollback on Zowe CLI V1, with latest versions of Z Open Editor and Zowe Explorer: problem disappeared!

Run setup for IBM User Build:
```
This log file is available at c:\Users\S0070188\AppData\Roaming\Code\logs\20220529T184045\exthost1\IBM.zopeneditor\userbuild-20220529-164408.log
Welcome to IBM User Build. Run User Build from the pop-up menu in an open editor with a COBOL, PL/I, or HLASM program.
Visit our IBM Docs to learn about IBM User Build here: https://www.ibm.com/docs/en/z-modernization-stack/2022.1.1?topic=code-building-cobol-pli-hlasm-programs-user-build.
=====================
Running setup for User Build...
Setup will upload files or folders that are defined in your ZAPP file or in .vscode/settings.json under "zopeneditor.userbuild.userSettings.additionalDependencies".
Checking the ZAPP file or workspace settings...
DBB ZAPP or workspace settings should be defined in either your repository's zapp.yaml, zapp.json or .vscode/settings.json file.
    command: /global/SYS9/DBB/bin/groovyz
    buildScriptArgs: --userBuild,--workspace /u/J070188/vscode,--application environ-techleads-sandbox/src,--hlq J070188.VSCODE.USERBUILD,--outDir /u/J070188/vscode/logs
    buildScriptPath: /u/J070188/DepotsGIT/zappbuild/build.groovy
    additionalDependencies: application-conf
---------------------
Checking user settings...
User settings should be defined in your VS Code or Theia Settings under zopeneditor.userbuild.userSettings
    localWorkspacePath: **Optional setting is missing**
    dbbWorkspace: /u/J070188/vscode
    dbbHlq: J070188.VSCODE.USERBUILD
    dbbLogDir: /u/J070188/vscode/logs
Using .gitattributes file at c:\Trv\DepotsGIT\environn-techleads-sandbox\.gitattributes
---------------------
The default Zowe CLI profile specified as TMVS in zopeneditor.zowe setting. Will try loading that.
Using Zowe profile TMVS
Using ssh profile TMVS-SSH

Checking if SSH profile TMVS-SSH can connect:
Connected successfully.

Identified the local root workspace path as c:\Trv\DepotsGIT\environn-techleads-sandbox
Running setup to build c:\Trv\DepotsGIT\environn-techleads-sandbox\src\COBOL\MINIMAL.cbl program.
Executing SSH command: 
mkdir -p '/u/J070188/vscode/environn-techleads-sandbox'
$ mkdir -p '/u/J070188/vscode/environn-techleads-sandbox'
$ exit $?
Executing SSH command: 
mkdir -p '/u/J070188/vscode/logs'
$ mkdir -p '/u/J070188/vscode/logs'
$ exit $?
Uploading additional dependencies...
Uploading c:\Trv\DepotsGIT\environn-techleads-sandbox\application-conf
Executing SSH command: 
mkdir -p '/u/J070188/vscode/environn-techleads-sandbox/application-conf'
$ mkdir -p '/u/J070188/vscode/environn-techleads-sandbox/application-conf'
$ exit $?
Finished running setup for User Build.
```

Run IBM User Build:
```
=====================
Running User Build...
Identified the local root workspace path as c:\Trv\DepotsGIT\environn-techleads-sandbox
---------------------
Checking the ZAPP file or workspace settings...
DBB ZAPP or workspace settings should be defined in either your repository's zapp.yaml, zapp.json or .vscode/settings.json file.
    command: /global/SYS9/DBB/bin/groovyz
    buildScriptArgs: --userBuild,--workspace /u/J070188/vscode,--application environ-techleads-sandbox/src,--hlq J070188.VSCODE.USERBUILD,--outDir /u/J070188/vscode/logs
    buildScriptPath: /u/J070188/DepotsGIT/zappbuild/build.groovy
    additionalDependencies: application-conf
---------------------
Checking user settings...
User settings should be defined in your VS Code or Theia Settings under zopeneditor.userbuild.userSettings
    localWorkspacePath: **Optional setting is missing**
    dbbWorkspace: /u/J070188/vscode
    dbbHlq: J070188.VSCODE.USERBUILD
    dbbLogDir: /u/J070188/vscode/logs
Using .gitattributes file at c:\Trv\DepotsGIT\environn-techleads-sandbox\.gitattributes
---------------------
The default Zowe CLI profile specified as TMVS in zopeneditor.zowe setting. Will try loading that.
Using Zowe profile TMVS
Using ssh profile TMVS-SSH

Checking if SSH profile TMVS-SSH can connect:
Connected successfully.

Executing SSH command: 
mkdir -p '/u/J070188/vscode/environn-techleads-sandbox/src/COBOL'
@@START OF COMMAND@@
$ mkdir -p '/u/J070188/vscode/environn-techleads-sandbox/src/COBOL'
$ exit $?
Uploading file c:\Trv\DepotsGIT\environn-techleads-sandbox\src\COBOL\MINIMAL.cbl to /u/J070188/vscode/environn-techleads-sandbox/src/COBOL/MINIMAL.cbl with encoding ibm-1147
Using ssh profile TMVS-SSH
Executing SSH command: 
chtag -tc ibm-1147 /u/J070188/vscode/environn-techleads-sandbox/src/COBOL/MINIMAL.cbl
$ chtag -tc ibm-1147 /u/J070188/vscode/environn-techleads-sandbox/src/COBOL/MI
$
$ environn-techleads-sandbox/src/COBOL/MIN                                    <IMAL.cbl
$ exit $?
Found a total of 0 local include files:
Checking if include files were updated since the last build...
Updating 0/0 local include files.
---------------------
Warning: Your workspace or ZAPP file settings for User Build do not contain a valid "--dependencyFile ${dependencyFile}" entry. If you use the latest version of dbb-zappbuild, add this setting to your build arguments to improve performance.
Executing SSH command: 
/global/SYS9/DBB/bin/groovyz /u/J070188/DepotsGIT/zappbuild/build.groovy --userBuild --workspace /u/J070188/vscode --application environ-techleads-sandbox/src --hlq J070188.VSCODE.USERBUILD --outDir /u/J070188/vscode/logs /u/J070188/vscode/environn-techleads-sandbox/src/COBOL/MINIMAL.cbl
$ /global/SYS9/DBB/bin/groovyz /u/J070188/DepotsGIT/zappbuild/build.groovy --u
$
$ 88/DepotsGIT/zappbuild/build.groovy --us                                    <erBuild --workspace /u/J070188/vscod
$
$ -userBuild --workspace /u/J070188/vscode                                    < --application environ-techleads-san
$
$ ode --application environ-techleads-sand                                    <box/src --hlq J070188.VSCODE.USERBUI
$
$ andbox/src --hlq J070188.VSCODE.USERBUIL                                    <D --outDir /u/J070188/vscode/logs /u
$
$ UILD --outDir /u/J070188/vscode/logs /u/                                    <J070188/vscode/environn-techleads-sa
$
$ /u/J070188/vscode/environn-techleads-san                                    <dbox/src/COBOL/MINIMAL.cbl
java.lang.SecurityException: java.lang.Exception - protected system package 'java.lang'
at java.lang.ClassLoader.checkClassName(ClassLoader.java:323)
at java.lang.ClassLoader.defineClassInternal(ClassLoader.java:377)
at java.lang.ClassLoader.defineClass(ClassLoader.java:359)
at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:154)
at java.net.URLClassLoader.defineClass(URLClassLoader.java:730)
at java.net.URLClassLoader.access$400(URLClassLoader.java:96)
at java.net.URLClassLoader$ClassFinder.run(URLClassLoader.java:1187)
at java.security.AccessController.doPrivileged(AccessController.java:770)
at java.net.URLClassLoader.findClass(URLClassLoader.java:605)
at org.codehaus.groovy.tools.RootLoader.oldFindClass(RootLoader.java:175)
at org.codehaus.groovy.tools.RootLoader.loadClass(RootLoader.java:147)
at java.lang.ClassLoader.loadClass(ClassLoader.java:872)
at java.lang.ClassLoader.defineClassImpl(Native Method)
at java.lang.ClassLoader.defineClassInternal(ClassLoader.java:398)
at java.lang.ClassLoader.defineClass(ClassLoader.java:359)
at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:154)
at java.net.URLClassLoader.defineClass(URLClassLoader.java:730)
at java.net.URLClassLoader.access$400(URLClassLoader.java:96)
at java.net.URLClassLoader$ClassFinder.run(URLClassLoader.java:1187)
at java.security.AccessController.doPrivileged(AccessController.java:770)
at java.net.URLClassLoader.findClass(URLClassLoader.java:605)
at org.codehaus.groovy.tools.RootLoader.oldFindClass(RootLoader.java:175)
at org.codehaus.groovy.tools.RootLoader.loadClass(RootLoader.java:147)
at java.lang.ClassLoader.loadClass(ClassLoader.java:872)
at java.lang.J9VMInternals.prepareClassImpl(Native Method)
at java.lang.J9VMInternals.prepare(J9VMInternals.java:303)
at java.lang.Class.getMethodHelper(Class.java:1247)
at java.lang.Class.getMethod(Class.java:1191)
at org.codehaus.groovy.tools.GroovyStarter.rootLoader(GroovyStarter.java:100)
at org.codehaus.groovy.tools.GroovyStarter.main(GroovyStarter.java:131)
** Build finished
$ exit $?
An error occurred: Error: Something went wrong executing SSH command /global/SYS9/DBB/bin/groovyz /u/J070188/DepotsGIT/zappbuild/build.groovy --userBuild --workspace /u/J070188/vscode --application environ-techleads-sandbox/src --hlq J070188.VSCODE.USERBUILD --outDir /u/J070188/vscode/logs /u/J070188/vscode/environn-techleads-sandbox/src/COBOL/MINIMAL.cbl
Looking for user build logs in "/u/J070188/vscode/logs".
Downloading logs to c:\Trv\DepotsGIT\environn-techleads-sandbox\logs
Successfully finished downloading of 0 log files to c:\Trv\DepotsGIT\environn-techleads-sandbox\logs.
```

There are some small issues with my IBM DBB configuration on z/OS-USS (being set up), but transfers between VS Code and z/OS-USS are fine, which was not the case with Zowe CLI V2.

Note however that there is a small waiting time after each of the `Using Zowe profile TMVS` and `Using ssh profile TMVS-SSH` lines.

So for me, there is indeed a problem of operation with Zowe CLI V2.

**Comment by FALLAI-Denis** [comment]
Hi,

For information, the blocking problem is still present when using Zowe V2, even after updating Zowe Explorer to version 2.0.3:

```
[2022/06/09 19:03:14.832] [DEBUG] [extension.js:32] Initialized logger from VSCode extension
[2022/06/09 19:03:15.875] [DEBUG] [extension.js:32] Zowe Explorer is using the team configuration file "zowe.config.json"
[2022/06/09 19:03:15.888] [DEBUG] [extension.js:32] Summary of team configuration files considered for Zowe Explorer: ["Path: C:\\Trv\\DepotsGIT\\environn-techleads-sandbox\\zowe.config.user.json: Not available ","Path: C:\\Trv\\DepotsGIT\\environn-techleads-sandbox\\zowe.config.json: Not available ","Path: C:\\Users\\S0070188\\.zowe\\zowe.config.user.json: Found, with the following defaults:{} ","Path: C:\\Users\\S0070188\\.zowe\\zowe.config.json: Found, with the following defaults:{\"ssh\":\"GMVS-SSH\",\"tso\":\"GMVS-TSO\",\"zftp\":\"GMVS-FTP\",\"zosmf\":\"GMVS\"} "]
[2022/06/09 19:03:18.438] [DEBUG] [extension.js:32] Initializing profiles with data set favorites.
[2022/06/09 19:03:18.512] [DEBUG] [extension.js:32] No data set favorites found.
[2022/06/09 19:03:26.544] [DEBUG] [extension.js:32] Initializing profiles with USS favorites.
[2022/06/09 19:03:26.642] [DEBUG] [extension.js:32] No USS favorites found.
[2022/06/09 19:03:35.357] [DEBUG] [extension.js:32] Initializing profiles with jobs favorites.
[2022/06/09 19:03:46.985] [DEBUG] [extension.js:32] [r [Error]: An error occurred collecting all configurations from the profile root directory "C:\Users\S0070188\.zowe\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.
	at new d (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:269457)
	at new c (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:261507)
	at new f (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:80679)
	at y.getCliProfileManager (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:52:68839)
	at y.getSchema (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:52:67470)
	at Function.getZoweVersion (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1121587)
	at Function.<anonymous> (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1020213)
	at Generator.next (<anonymous>)
	at s (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1014409)
	at processTicksAndRejections (node:internal/process/task_queues:96:5)] {
  mDetails: {
    msg: 'An error occurred collecting all configurations from the profile root directory "C:\\Users\\S0070188\\.zowe\\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.',
    additionalDetails: [r [Error]: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.
    	at Function.crashInTeamConfigMode (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:97792)
    	at Function.getAllProfileDirectories (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:96536)
    	at f.collectAllConfigurations (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:286781)
    	at new d (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:269211)
    	at new c (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:261507)
    	at new f (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:72:80679)
    	at y.getCliProfileManager (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:52:68839)
    	at y.getSchema (c:\Users\S0070188\.vscode\extensions\zowe.vscode-extension-for-zowe-2.0.3\out\src\extension.js:52:67470)
    	at Function.getZoweVersion (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1121587)
    	at Function.<anonymous> (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1020213)
    	at Generator.next (<anonymous>)
    	at s (c:\Users\S0070188\.vscode\extensions\ibm.zopeneditor-2.0.3\dist\extension.js:5:1014409)
    	at processTicksAndRejections (node:internal/process/task_queues:96:5)] {
      mDetails: [Object],
      mMessage: 'Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.'
    }
  },
  mMessage: 'An error occurred collecting all configurations from the profile root directory "C:\\Users\\S0070188\\.zowe\\profiles". Please supply the configurations on the profile manager constructor parameters OR initialize the profile manager environment. Details: Profile IO Error: A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.'
}
[2022/06/09 19:03:47.849] [DEBUG] [extension.js:32] File was saved -- determining whether the file is a USS file or Data set.
 Comparing (case insensitive) c:\Users\S0070188\AppData\Roaming\Code\User\settings.json against directory C:\Trv\DepotsGIT\Temp\temp\_D_andC:\Trv\DepotsGIT\Temp\temp\_U_
[2022/06/09 19:03:47.853] [DEBUG] [extension.js:32] File c:\Users\S0070188\AppData\Roaming\Code\User\settings.json is not a data set or USS file 
```

**Comment by FALLAI-Denis** [comment]
Hi @phaumer 

The problem is still present in the combination Z Open Editor 2.1.0, Zowe Explorer 2.1.0, and Zowe CLI "zosmf" V2, (internal version 7.1.2).

No problem with Zowe CLI "zosmf" V1.

This problem becomes critical because we are in the decision phase to choose (or not) the IBM Dependency Based Build product.

**Comment by FALLAI-Denis** [comment]
For information, the problem also occurs (systematically) on workstations directly installed with Zowe CLI "zosmf" V2.

Uninstalling Zowe CLI "zosmf" V2, followed by installing Zowe CLI "zosmf" V1, with the same installation of node.js/npm does not resolve the issue.

To solve the problem, you have to use Zowe CLI V1 with a fresh installation of node.js / npm, (and deleting the .zowe folder in the user's home directory, but this action is not enough).

So I guess something is saved by Zowe CLI "zosmf" V2 in node.js/npm folder, which does not go away with uninstalling Zowe CLI "zosmf" V2 and is not changed by installing Zowe CLI "zosmf" V1.

For the moment the origin of the problem is not clearly identified.

**Comment by phaumer** [comment]
Added a fix in v2.1.1.
